Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -89,26 +89,29 @@ public struct ConfigSetting: BazelTarget {
     }
 }
 
-// https://bazel.build/versions/master/docs/be/objective-c.html#objc_framework
-public struct ObjcFramework: BazelTarget {
+// https://github.com/bazelbuild/rules_apple/blob/818e795208ae3ca1cf1501205549d46e6bc88d73/doc/rules-general.md#apple_static_framework_import
+public struct AppleStaticFrameworkImport: BazelTarget {
     let name: String // A unique name for this rule.
     let frameworkImports: AttrSet<[String]> // The list of files under a .framework directory which are provided to Objective-C targets that depend on this target.
 
     var acknowledged: Bool {
         return true
     }
 
-    // objc_framework(
+    // FIXME: provide an API for apple_dynamic_framework_import.
```

Comment:
Can you file an issue for this and put a link to that issue in the comments here?

Reply:
I just moved these comments from

https://github.com/pinterest/PodToBUILD/blob/7524dd21cd72067ee15b0564da90aed0be79d3ae/Sources/PodToBUILD/ObjcLibrary.swift#L121-L124.

Reply:
Got it. Seems like the FIXME isn't really needed if we are intentionally declaring that all frameworks are static.